cmd/security-agent/subcommands/check/command.go

```diff
@@ -140,20 +135,13 @@ func RunCheck(log log.Component, config config.Component, _ secrets.Component, s
 	var resolver compliance.Resolver
 	if checkArgs.overrideRegoInput != "" {
 		resolver = newFakeResolver(checkArgs.overrideRegoInput)
-	} else if flavor.GetFlavor() == flavor.ClusterAgent {
-		resolver = compliance.NewResolver(context.Background(), compliance.ResolverOptions{
-			Hostname:           hname,
-			DockerProvider:     compliance.DefaultDockerProvider,
-			LinuxAuditProvider: compliance.DefaultLinuxAuditProvider,
-			KubernetesProvider: complianceKubernetesProvider,
-			StatsdClient:       statsdClient,
-		})
 	} else {
 		resolver = compliance.NewResolver(context.Background(), compliance.ResolverOptions{
 			Hostname:           hname,
 			HostRoot:           os.Getenv("HOST_ROOT"),
```

Comment:
This was not set before for the cluster agent, but if the env var is not there it will be "" which is the default